nodeadm/internal/aws/ecr/ecr.go

```diff
@@ -19,7 +21,14 @@ func GetAuthorizationToken(awsRegion string) (string, error) {
 		return "", err
 	}
 	ecrClient := ecr.NewFromConfig(awsConfig)
-	token, err := ecrClient.GetAuthorizationToken(context.Background(), &ecr.GetAuthorizationTokenInput{})
+	var token *ecr.GetAuthorizationTokenOutput
+	err = util.RetryExponentialBackoff(3, 2*time.Second, func() error {
+		token, err = ecrClient.GetAuthorizationToken(context.Background(), &ecr.GetAuthorizationTokenInput{})
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
nit: you can collapse these :) 
```suggestion
		return err
```

Reply:
Revised!